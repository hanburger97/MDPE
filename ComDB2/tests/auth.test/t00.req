put password 'password1' for 'user1'
put password 'password2' for 'user2'
grant op to user1
#only op can turn on authentication
put authentication on
