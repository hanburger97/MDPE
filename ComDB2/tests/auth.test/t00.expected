[put authentication on] failed with rc -3 User does not have OP credentials
